com.github.WhatsApp-For-Linux.yml

```diff
@@ -0,0 +1,166 @@
+app-id: com.github.WhatsApp-For-Linux
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+command: whatsapp-for-linux
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+# - --device=dri # OpenGL rendering
+  - --device=all
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --own-name=com.github.whatsapp-for-linux
```

Comment:
This is implicit based on the app-id, BUT, see the comment about the app-id so it might be needed after all.